pkg/model/devrc.go

```diff
@@ -0,0 +1,187 @@
+package model
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"strings"
+	"time"
+
+	"github.com/okteto/okteto/pkg/log"
+	yaml "gopkg.in/yaml.v2"
+	apiv1 "k8s.io/api/core/v1"
+)
+
+// Dev represents a development container
+type DevRC struct {
+	Labels               Labels                `json:"labels,omitempty" yaml:"labels,omitempty"`
+	Annotations          Annotations           `json:"annotations,omitempty" yaml:"annotations,omitempty"`
+	Context              string                `json:"context,omitempty" yaml:"context,omitempty"`
+	Namespace            string                `json:"namespace,omitempty" yaml:"namespace,omitempty"`
+	Environment          Environment           `json:"environment,omitempty" yaml:"environment,omitempty"`
+	Secrets              []Secret              `json:"secrets,omitempty" yaml:"secrets,omitempty"`
+	Sync                 Sync                  `json:"sync,omitempty" yaml:"sync,omitempty"`
+	Resources            ResourceRequirements  `json:"resources,omitempty" yaml:"resources,omitempty"`
+	PersistentVolumeInfo *PersistentVolumeInfo `json:"persistentVolume,omitempty" yaml:"persistentVolume,omitempty"`
+	Timeout              time.Duration         `json:"timeout,omitempty" yaml:"timeout,omitempty"`
+}
+
+// Get returns a Dev object from a given file
+func GetRc(devPath string) (*DevRC, error) {
+	b, err := ioutil.ReadFile(devPath)
+	if err != nil {
+		return nil, err
+	}
+
+	dev, err := ReadRC(b)
+	if err != nil {
+		log.Warning("ignoring developer overwrites defined in %s: %s", devPath, err.Error())
```

Comment:
I think we should fail this instead of ignoring, since this will result in an `okteto up` with unexpected values.